### DISCUSSION TITLE
ADIF import - Operator Name

### DISCUSSION THREAD

**Discussion by oe3ide** [magicbug/Cloudlog#990] [General]
Hi,
currently the adif-import takes the operator-name from the adif. But if there is no operator-name in the imported adif, operator-name keeps blank. Would be better to take the user's callsign as operator-name.

An admin-option like "always use login-callsign as operator-name on import" would be great!

Reason:
We try to use cloudlog for our next S.E.S. We expect around 5-10 operators. Some logging in their own logging-software (uploading adifs), some logging direct in cloudlog. As S.E.S manager I would like to be sure, that I have the correct operator name on adif-imports.

73s
Ernst, OE3IDE

**Comment by AndreasK79** [comment]
Hi.

What about the operator callsign? That should be set, and will currently be imported now.
It is also displayed when looking at a QSO in Cloudlog.

Andreas.

**Comment by oe3ide** [comment]
Hi Andreas,
yes, correct. I tested it, if the operator callsign is set correctly in the adif, it will be imported correct.

BUT, as manager I do not know, which logging-software is used by my friends, and I do not know if it is set correct in their logger.

73 de Ernst
OE3IDE

**Reply by AndreasK79** [reply]
Ah, now I understand. I read again what you wrote. It should be doable. I'll write it down in my list and see if I can figure out something.

Andreas.

**Comment by AndreasK79** [comment]
#1013 added.
